+siibra/atlas.m

```diff
@@ -0,0 +1,8 @@
+function atlas = atlas(atlas_name)
+% Construct atlas by name
```

Comment:
A good practice for user-facing functions is to use [function argument validation](https://www.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html) to specify the accepted type and other constraints for valid input arguments. It will do sensible things, e.g. can specify the String type here and it would auto-accept (and convert) char inputs.

Reply:
Very nice! I will add them :)